### PR TITLE
Issue #8: Request Controller

### PR DESCRIPTION
**Pull request by jtingling**
### What this PR does (required):
- Create, read, update operations on Request model for dog sitting.

### Any information needed to test this feature (required):
- Register two users.
- Set a environment variable `TOKEN` with a encoded JWT token of the dog owner;
- Re-assign `userId` and `sitterId` values to the `id` values of the  users you registered in `models.spec.js`.
- On the command line, run `npm run testRequets`.

### CODE REVIEW

File: server/controllers/request.js
Comment (ezeilo-su) [inline server/controllers/request.js:5]: You need to return a `400` if this field is missing

File: server/controllers/request.js
Comment (ezeilo-su) [inline server/controllers/request.js:20]: Please validate the request body and returna `400` if any required filed is missing

File: server/controllers/request.js
Comment (ezeilo-su) [inline server/controllers/request.js:16]: You can use `await` here since the function is an `async` type

File: server/controllers/request.js
Comment (ezeilo-su) [inline server/controllers/request.js:33]: Use `await` with an `async` function

File: server/controllers/request.js
Comment (ezeilo-su) [inline server/controllers/request.js:48]: Same as the comments above

File: server/models/Request.js
Comment (ezeilo-su) [inline server/models/Request.js:6]: This should reference the `User` model

File: server/models/Request.js
Comment (ezeilo-su) [inline server/models/Request.js:10]: Add a `ref` to the `User` model

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Good work with this one. A few comments to address, though.

**Review by ezeilo-su — COMMENTED** [review]
Thank you for making the requested changes. Please wait for the team to approve before you can merge.
Happy coding!

**Review by moffatethan — APPROVED** [review]

**Pull request merged**
